src/main/java/it/pagopa/transactions/model/Transaction.java

```diff
@@ -0,0 +1,35 @@
+package it.pagopa.transactions.model;
```

Comment:
what do you think if we put all the `aggregate` and `valueObjet` in a different package? like domain?

Reply:
I'm ok with that, would it be everything besides `SoapEnvelope`? If it boils down to that we could even rename `model` → `domain` and put `SoapEnvelope` in some other package (just saying, I don't feel like `SoapEnvelope` belongs to "model")?

Reply:
correct!